### PR TITLE
modify table appear animation

### PR DESCRIPTION
**Pull request by naoyashiga**
BTNavigationDropdownMenu is good, especially rotate arrow icon animation.

I think it is better to fix table appear animation.

Please review.

### CODE REVIEW

File: Source/BTNavigationDropdownMenu.swift [hunk 1]
```diff
@@ -203,15 +203,19 @@ public class BTNavigationDropdownMenu: UIView {
         
         // Animation
         self.tableView.frame.origin.y = -CGFloat(self.items.count) * self.configuration.cellHeight - 300
-        UIView.animateWithDuration(self.configuration.animationDuration, animations: {
-            self.tableView.frame.origin.y = CGFloat(-300)
-            self.tableView.contentOffset = CGPointMake(0, -self.configuration.bounceOffset);
```
Comment (naoyashiga) [inline Source/BTNavigationDropdownMenu.swift]: Do you want to do bounce animation?

File: Source/BTNavigationDropdownMenu.swift [hunk 2]
```diff
@@ -203,15 +203,19 @@ public class BTNavigationDropdownMenu: UIView {
...
-            }, completion: { _ in
-                UIView.animateWithDuration(self.configuration.animationDuration/3, animations: {
-                    self.tableView.contentOffset = CGPointMake(0, 0)
-                })
-        })
+        
+        UIView.animateWithDuration(
+            self.configuration.animationDuration * 1.5,
+            delay: 0,
+            usingSpringWithDamping: 0.7,
```
Comment (naoyashiga) [inline Source/BTNavigationDropdownMenu.swift]: Animation using spring is the best way to bounce.

File: Demo/Demo/ViewController.swift [hunk 3]
```diff
@@ -28,7 +28,7 @@ class ViewController: UIViewController {
         menuView.cellTextLabelColor = UIColor.whiteColor()
         menuView.cellTextLabelFont = UIFont(name: "Avenir-Heavy", size: 17)
         menuView.arrowPadding = 15
-        menuView.animationDuration = 0.3
```
Comment (naoyashiga) [inline Demo/Demo/ViewController.swift]: Duration is too short. We don't find bounce animation. More slowly is better.

File: Source/BTNavigationDropdownMenu.swift [hunk 4]
```diff
@@ -222,7 +226,7 @@ public class BTNavigationDropdownMenu: UIView {
         self.backgroundView.alpha = self.configuration.maskBackgroundOpacity
         
         // Animation
-        UIView.animateWithDuration(self.configuration.animationDuration, delay: 0.15, options: UIViewAnimationOptions.TransitionNone, animations: {
```
Comment (naoyashiga) [inline Source/BTNavigationDropdownMenu.swift]: Why do you have this delay?

I think delay should be zero because animation is faster.

Comment (PhamBaTho) [inline Source/BTNavigationDropdownMenu.swift]: The spring animation looks great. But If I use spring animation, how can I change bounce offset of spring animation? And how can I make use of existing `bounceOffset` property?

Comment (naoyashiga) [inline Source/BTNavigationDropdownMenu.swift]: I agreed with you. Think deeply, I can't arrange bounce offset....

**Comment by PhamBaTho** [comment]
Yes. I decided to drop unnecessary bounceOffset property and use spring animation instead. This makes animation smoother. Thanks!

**Comment by naoyashiga** [comment]
@PhamBaTho Thanks for merge :+1:

**Pull request merged**